examples/launcher/src/components/Engine.jsx

```diff
@@ -204,6 +192,16 @@ class Engine extends React.Component {
       this.blur();
     }
 
+    postViewportMessage() {
```

Comment:
This code is still duplicated [here](https://github.com/exokitxr/exokit/pull/1073/files#diff-818c6db2ba21b9def5bafb0e208c1111R26).

Reply:
c86b1c4